example/src/services/PlaybackService.ts

```diff
@@ -25,14 +25,12 @@ export async function PlaybackService() {
 
   TrackPlayer.addEventListener(Event.RemoteJumpForward, async (event) => {
     console.log('Event.RemoteJumpForward', event);
-    const position = (await TrackPlayer.getPosition()) + event.interval;
-    TrackPlayer.seekTo(position);
+    TrackPlayer.seekBy(event.interval);
```

Comment:
😍😍

Reply:
The small things : )

(Would be nice to go even further and have an option where we could just have the native side handle these cases completely – so we wouldn't have to add event listeners for `Event. RemoteJumpForward` etc.)